engines/instance_verification/lib/instance_verification/providers/example.rb

```diff
@@ -49,4 +49,8 @@ def payg_billing_code?(iid, identifier)
     return true if (identifier.casecmp('sles').zero? && instance_billing_info[:billing_product] == SLES_PRODUCT_IDENTIFIER)
     return true if (identifier.casecmp('sles_sap').zero? && SLES4SAP_PRODUCT_IDENTIFIER.include?(instance_billing_info[:marketplace_code]))
   end
+
+  def basic?
+    false
+  end
```

Comment:
We should probably have some comment here explaining what this is intended to do. The term  `basic` means something to us (SUSE) because we publish so called `basic` images in Azure and those images have no support entitlements but this meaning is no obvious to any 3rd party CSP using the Public Cloud features in  that are part of RMT.